test/decorators.py

```diff
@@ -23,6 +23,20 @@
 from .unit.mock.fake_runtime_service import FakeRuntimeService
 
 
+def production_only(func):
+    """Decorator that runs a test only on production services."""
+
+    @wraps(func)
+    def _wrapper(self, *args):
+        if "dev" in os.environ.get("QISKIT_IBM_URL", ""):
```

Comment:
Couldn't find another way to check if the environment was staging - open to suggestions

Reply:
I like the idea of adding this as a decorator. I don't have any insight regarding your method of checking. I am just wondering if the URL is necessarily taken from the environment, because I think I recall that the URL can be given directly in the interface. But I may be wrong.

Reply:
Ah you're right, i'll have to find another way to do this

Reply:
Updated to use the service dependency url instead of an environment variable